packages/api/package.json

```diff
@@ -19,10 +19,12 @@
   },
   "dependencies": {
     "@babel/runtime": "7.16.7",
+    "bignumber.js": "9.0.2",
```

Comment:
I added this function because json-bigint has a bug https://github.com/sidorares/json-bigint/issues/63 and I would like to have all numbers <= Number.MAX_SAFE_INTEGER as the standard type. You are not able to use BigNumber directly as React value and it will throw an exception when you try to render it.